planck-c/engine.c

```diff
@@ -151,7 +151,7 @@ JSValueRef get_value(JSContextRef ctx, char *namespace, char *name) {
 
     // printf("get_value: '%s'\n", namespace);
     char *ns_tmp = strdup(namespace);
-    char *saveptr;
+    char *saveptr = NULL;
```

Comment:
Could you explain the background to what's going on here?

Reply:
The compiler is concerned that this value might be used without being initialized. In fact, this is not true, but the compiler is not smart enough to deduce this. So an easy fix is to simply initialize the variable.